content/en/containers/kubernetes/prometheus.md

```diff
@@ -216,6 +216,16 @@ With Prometheus Autodiscovery, the Datadog Agent is able to detect native Promet
 
 ### Configuration
 
+It's recommended to first check which pods and services have the `prometheus.io/scrape=true` annotation before enabling this feature. This can be done with commands like:
```

Comment:
```suggestion
It's recommended to first check which pods and services have the `prometheus.io/scrape=true` annotation before enabling this feature. This can be done with the following commands:
```